pandas/errors/__init__.py

```diff
@@ -91,6 +91,42 @@ class PerformanceWarning(Warning):
     """
 
 
+class PandasChangeWarning(Warning):
```

Comment:
Not in love with this name, but `PandasDeprecationWarning` is taken below. Bikeshedding here is most welcome!